src/Pages/jinahhong/Login/LoginH.scss

```
@@ -0,0 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Lobster&display=swap");
+.login_body {
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+
+  .center {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid rgba(var(--b6a, 219, 219, 219), 1);
+    border-radius: 1px;
+    padding: 10px 0;
+    width: 350px;
+    height: 366px;
```

css 속성 순서를 Box Model Properties (width, height, margin, padding) 순서로 수정하면 더 좋을 것 같아요~
```suggestion
  .center {
    width: 350px;
    height: 366px;
    padding: 10px 0;
    display: flex;
    flex-direction: column;
    align-items: center;
    border: 1px solid rgba(var(--b6a, 219, 219, 219), 1);
    border-radius: 1px;
    
```